python/lsst/meas/astrom/astrometry.py

```diff
@@ -215,25 +218,34 @@ def solve(self, sourceCat, bbox, initWcs, filterName=None, calib=None, exposure=
                 res = tryRes
                 break
 
-            self.log.logdebug(
+            self.log.info(
                 "Fit WCS iter %s: %s matches; median scatter = %g arcsec" % \
                     (i, len(tryRes.matches), tryRes.scatterOnSky.asArcseconds()),
             )
 
-            if res is not None and not self.config.forceKnownWcs:
```

Comment:
I have completely removed the convergence criteria and just iterate config.maxIter times. This part needs more testing in order to understand if it is robust enough.